libs/shared-web/src/utils/auth-selectors/selectShouldInitializeLaunchDarkly.tsx

```diff
@@ -0,0 +1,25 @@
+import { selectHasUserCompletedSignUp } from './selectHasUserCompletedSignUp';
+import { checkShouldUseLaunchDarkly } from '../checkShouldUseLaunchDarkly';
+import { UserContext } from '../../providers/AuthProvider';
+
+/** Determine if LaunchDarkly should be initialized based on the current auth context */
+export function selectShouldInitializeLaunchDarkly(userCtx: UserContext): boolean {
+  const { isLoggedIn, currentOrganization } = userCtx;
+  // don't show fallback if LaunchDarkly isn't enabled
+  if (!checkShouldUseLaunchDarkly()) {
+    return false;
+  }
+
+  // enable feature flags for unauthenticated areas of the app
+  if (!isLoggedIn) {
```

Comment:
Do we have to stick to this restriction for now? There might be a case in the future where we want to feature flag public pages such as sign-in or sign-up.

Reply:
This check is what prevents it from blocking on public pages!